scripts/models.py

```diff
@@ -0,0 +1,251 @@
+
+
+class RoleType(str, Enum):
+    Author = "Author"
+    Conceptualization = "Conceptualization"
+    ContactPerson = "ContactPerson"
+    DataCollector = "DataCollector"
+    DataCurator = "DataCurator"
+    DataManager = "DataManager"
+    FormalAnalysis = "FormalAnalysis"
+    FundingAcquisition = "FundingAcquisition"
+    Investigation = "Investigation"
+    Maintainer = "Maintainer"
+    Methodology = "Methodology"
+    Producer = "Producer"
+    ProjectLeader = "ProjectLeader"
+    ProjectManager = "ProjectManager"
+    ProjectMember = "ProjectMember"
+    ProjectAdministration = "ProjectAdministration"
+    Researcher = "Researcher"
+    Resources = "Resources"
+    Software = "Software"
+    Supervision = "Supervision"
+    Validation = "Validation"
+    Visualization = "Visualization"
+    Funder = "Funder"
+    Sponsor = "Sponsor"
+    StudyParticipant = "StudyParticipant"
+    Other = "Other"
+
+
+class Relation(str, Enum):
+    IsCitedBy = "IsCitedBy"
+    Cites = "Cites"
+    IsSupplementTo = "IsSupplementTo"
+    IsSupplementedBy = "IsSupplementedBy"
+    IsContinuedBy = "IsContinuedBy"
+    Continues = "Continues"
+    Describes = "Describes"
+    IsDescribedBy = "IsDescribedBy"
+    HasMetadata = "HasMetadata"
+    IsMetadataFor = "IsMetadataFor"
+    HasVersion = "HasVersion"
+    IsVersionOf = "IsVersionOf"
+    IsNewVersionOf = "IsNewVersionOf"
+    IsPreviousVersionOf = "IsPreviousVersionOf"
+    IsPartOf = "IsPartOf"
+    HasPart = "HasPart"
+    IsReferencedBy = "IsReferencedBy"
+    References = "References"
+    IsDocumentedBy = "IsDocumentedBy"
+    Documents = "Documents"
+    IsCompiledBy = "IsCompiledBy"
+    Compiles = "Compiles"
+    IsVariantFormOf = "IsVariantFormOf"
+    IsOriginalFormOf = "IsOriginalFormOf"
+    IsIdenticalTo = "IsIdenticalTo"
+    IsReviewedBy = "IsReviewedBy"
+    Reviews = "Reviews"
+    IsDerivedFrom = "IsDerivedFrom"
+    IsSourceOf = "IsSourceOf"
+    IsRequiredBy = "IsRequiredBy"
+    Requires = "Requires"
+    Obsoletes = "Obsoletes"
+    IsObsoletedBy = "IsObsoletedBy"
+
+
+class IdentifierType(str, Enum):
+    doi = "DOI"
+    orcid = "ORCID"
+    ror = "ROR"
+    dandi = "DANDI"
+    rrid = "RRID"
+    none = "No identifier prefix"
+
+
+class PropertyValue(BaseModel):
+    maxValue: float = None
+    minValue: float = None
+    unitCode: Union[str, AnyUrl] = None
+    unitText: str = None
+    value: Union[bool, float, str, int, List[Union[bool, float, str, int]]] = None
+    valueReference: PropertyValue = None
+    propertyID: Union[str, AnyUrl] = None
+
+
+PropertyValue.update_forward_refs()
+
+
+class Identifier(BaseModel):
+    identifier: Union[str, AnyUrl, PropertyValue]
+
+
+class Contributor(BaseModel):
+    identifier: Identifier = None
+    name: str
+    email: EmailStr = None
+    url: AnyUrl = None
+    roleName: List[RoleType]
+    includeInCitation: bool = True
+    awardNumber: str = None
+
+
+class Person(Contributor):
+    affiliation: List[str]
```

Comment:
This says the field is `affiliation`, but on the converted dandisets you have the field `affiliations`.